lib/claim_letters/claim_letter_downloader.rb

```diff
@@ -71,8 +71,15 @@ def filter_letters(document)
       document.marshal_dump
     end
 
+    def filter_boa_letters(document)
+      return false if document[:doc_type] == '27' && Time.zone.today - document[:received_at] < 2
```

Comment:
I noticed this line: `# TODO: (rare) Handle nil received_at` below. Do you think that could happen here? Should there be any error handling here?